src/Routing/Router.php

```diff
@@ -594,6 +594,8 @@ public static function url($url = null, $full = false)
                 unset($url['_ssl']);
             }
 
+            $url = static::_applyUrlFilters($url);
```

Comment:
I wonder if this is the kind of behavior change that will catch people off guard when they are updating.